rosidl_generator_cpp/test/test_traits.cpp

```diff
@@ -0,0 +1,76 @@
+// Copyright 2019 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+#include "rosidl_generator_cpp/msg/empty.hpp"
+#include "rosidl_generator_cpp/msg/string.hpp"
+#include "rosidl_generator_cpp/srv/complex_type_zero_fill.hpp"
```

Comment:
I just noticed that these tests need to be updated after rebasing. Since https://github.com/ros2/rosidl/pull/397, we are using a different set of test interfaces, ie. `ComplexTypeZeroFill.msg` and `String.msg` do not exist.

Reply:
Rebased and updated in 4fc81a9